corehq/apps/domain/auth.py

```diff
@@ -77,6 +76,12 @@ def get_username_and_password_from_request(request):
                 # decode password submitted from mobile app login
                 password = decode_password(password)
 
+                try:
+                    password = password.decode('utf-8')
```

Comment:
Should also decode the username here too. The encoding will be the same for both.

Reply:
updated